components/utils/src/site.rs

```diff
@@ -86,11 +86,18 @@ mod tests {
         assert!(res.is_err());
     }
 
+    #[test]
+    fn reading_analytics_empty_text() {
+        let (word_count, reading_time) = get_reading_analytics("  ");
+        assert_eq!(word_count, 0);
+        assert_eq!(reading_time, 0);
```

Comment:
shouldn't that be one according to your PR comment?

Reply:
Didn't realize Medium allows a title-only post. Just tested in Medium and it is 1 min read, although I think 0 min read makes more sense. What do you prefer?

Reply:
0 makes more sense yes.